src/screens/TopicPageScreen/index.js

```diff
@@ -63,7 +63,7 @@ const TopicPageScreen = (props) => {
   const [topicId, setTopicId] = React.useState('');
   const [isFollow, setIsFollow] = React.useState(params.isFollowing);
   const [topicDetail, setTopicDetail] = React.useState({});
-  const [memberCount, setMemberCount] = React.useState(0);
+  const [memberCount, setMemberCount] = React.useState(params.memberCount);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from initializing `memberCount` with 0 to `params.memberCount` could potentially cause issues if `params.memberCount` is undefined or not a number. It would be safer to add a check for this.
```diff
- const [memberCount, setMemberCount] = React.useState(params.memberCount);
+ const [memberCount, setMemberCount] = React.useState(Number.isInteger(params.memberCount) ? params.memberCount : 0);
```
This way, we ensure that `memberCount` always starts as a number and prevent potential type-related bugs later in the code.


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
code rabbit please, that's not the way we handle state 🥲